crates/component-macro/tests/codegen.rs

```diff
@@ -1,21 +1,21 @@
 macro_rules! gentest {
-    ($id:ident $path:tt) => {
+    ($id:ident $name:tt $path:tt) => {
         mod $id {
             mod normal {
-                wasmtime::component::bindgen!($path);
-            }
-            mod async_ {
-                wasmtime::component::bindgen!({
-                    path: $path,
-                    async: true,
-                });
-            }
-            mod tracing {
-                wasmtime::component::bindgen!({
-                    path: $path,
-                    tracing: true,
-                });
+                wasmtime::component::bindgen!($name in $path);
             }
+            // mod async_ {
```

Comment:
Should we add a `TODO` comment above explaining why these are commented out?

Reply:
Whoops thanks for catching I forgot to uncomment.